mobile.js

```diff
@@ -0,0 +1,58 @@
+function saveFormDataToLocalStorage() {
+  const nameInput = document.querySelector('input[name="name"]');
+  const emailInput = document.querySelector('.email-box');
```

Comment:
- [ ] I can see that you have created another mobile.js with the code of this file and using ES6 arrow syntax but the problem is that this file still exists here, what he meant was to take this file into modules and then suggested to export it in index.js.
- [ ] I will recommend you to remove this file from here, as you taking the contents of this file in the modules directory. having the same content in two separate files is against DRY principle so please consider fixing it.

Reply:
Hi @Zuhaib042 Sorry I removed the file but I did not commit the change.